docs/quantum_chess/concepts.ipynb

```diff
@@ -18,7 +18,7 @@
    },
    "outputs": [],
    "source": [
-    "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# @title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
```

Comment:
will this break the devsite parsing?

Reply:
I don't think so but other notebooks don't have a space, so changed to that.